components/board/client.go

```diff
@@ -253,6 +265,125 @@ func (dic *digitalInterruptClient) RemoveCallback(c chan Tick) {
 	panic(errUnimplemented)
 }
```

Comment:
Should we get rid of these functions now? Lines 244 -> 252

Reply:
I can't remove since these are in the digital interrupt interface. I could implement them but I don't see why they would ever be used client side.

Reply:
discussed in person: I will remove them from interface and instead make them helper functions in future PR.